test/azure/generated/AzureCompositeModelClient/models/mappers.ts

```diff
@@ -9,11 +9,12 @@
  */
 
 import {CloudErrorMapper, BaseResourceMapper } from "ms-rest-azure-js";
+import * as msRest from "ms-rest-js";
```

Comment:
Isn't it better for webpack if you did `import { CompositeMapper} from "ms-rest-js";`?

Reply:
It's possible it will save webpack time when tree shaking, but webpack is still able to shake `import * as msRest` imports, as long as you don't pass the `msRest` object off to a function or something like that (like we do with `Mappers`)

I think I was mainly thinking about name collisions--even though we're realistically already in a bit of a precarious spot with that.